sklearn/covariance/_robust_covariance.py

```diff
@@ -66,7 +66,7 @@ def c_step(X, n_support, remaining_iterations=30, initial_estimates=None,
         If int, random_state is the seed used by the random number generator;
         If RandomState instance, random_state is the random number generator;
         If None, the random number generator is the RandomState instance used
-        by `np.random`.
+        by `np.random`. See :term: `Glossary <random_state>`.
```

Comment:
apply the same changes here (and below) maybe

i.e. remove the long description and add comment about reproducible results

Reply:
done :)